docs/man_pages/project/configuration/generate-icon.md

```diff
@@ -0,0 +1,34 @@
+generate icon
+==========
+
+Usage | Synopsis
+------|-------
+General | `$ appbuilder generate icon <Path>` [--force]
+
+Generates icons from a single high-resolution image and saves them to the current project
+
+<% if(isConsole) { %>
+<% if(isMobileWebsite)  { %>
+WARNING: This command and its extended commands are not applicable to mobile website projects. To view the complete help for this command, run `$ appbuilder help generate`
+<% } %>
+<% } %>
+<% if((isConsole && isCordova && isNativeScript) || isHtml) { %>
```

Comment:
Not sure if this check will actually work, a project cannot be both a Cordova and a NativeScript one. Perhaps go with `((isConsole && (isCordova || isNativeScript)) || isHtml)`